dist/debian/rules

```diff
@@ -45,17 +45,20 @@ DEB_SC_BUILDDIR ?= builddir
 
 # see mconfig for options
 # set environment variables to disable options
-# NOALL, NONETWORK, NOSECCOMP
+# NOALL, NONETWORK, NOSECCOMP, LIBSUBID
 SC_OPTIONS =
 ifdef DEB_NOALL
-SC_OPTIONS = --without-network --without-seccomp
+SC_OPTIONS = --without-network --without-seccomp --without-libsubid
 else
 ifdef DEB_NONETWORK
 SC_OPTIONS += --without-network
 endif
 ifdef DEB_NOSECCOMP
 SC_OPTIONS += --without-seccomp
 endif
+ifdef DEB_LIBSUBID
```

Comment:
This new option would need to be documented in dist/debian/DEBIAN_PACKAGE.md.

Reply:
Docs added